src/servers/src/grpc/flight/stream.rs

```diff
@@ -146,7 +145,7 @@ mod test {
         let recordbatches = RecordBatches::try_new(schema.clone(), vec![recordbatch.clone()])
             .unwrap()
             .as_stream();
-        let mut stream = FlightRecordBatchStream::new(recordbatches);
+        let mut stream = FlightRecordBatchStream::new(recordbatches, 0);
```

Comment:
I don't like the magic number 0.

Reply:
This is test code. Or how do you think of replacing it with random one?

Reply:
Ok, let's leave it.